modules/rds/main.tf

```diff
@@ -25,13 +56,19 @@ resource "aws_db_instance" "default" {
   instance_class             = var.instance_class
   username                   = var.username
   password                   = var.password
-  backup_retention_period    = 1
+  backup_retention_period    = var.backup_retention_period
```

Comment:
Added some more variables to allow more flexibility with settings.

Reply:
Dont feel like you have to go all out with the customizations, for the initial deployment. I like the phrase "Make it work, then make it better" ™️  for things like this.  Still for simple things like this its totally fine to fix and address them as you go.